src/systems/ec/implementations/vanilla/packages/ec-preset-full/_imports.scss

```diff
@@ -58,6 +58,7 @@
 // Molecules
 @import '@ecl/ec-component-hero-banner/ec-component-hero-banner';
 @import '@ecl/ec-component-breadcrumb/ec-component-breadcrumb';
+@import '@ecl/ec-component-card/ec-component-card';
```

Comment:
Don't forget to also add the component to the preset's package.json dependencies